datahub-classify/src/datahub_classify/infotype_helper.py

```diff
@@ -30,16 +30,26 @@
 )
 
 logger = logging.getLogger(__name__)
-spacy_model_name = "en_core_web_sm"
-try:
-    nlp_english = spacy.load(spacy_model_name)
-except OSError:
-    logger.info("Downloading language model for the spaCy")
-    from spacy.cli import download
 
-    download(spacy_model_name)
-    nlp_english = spacy.load(spacy_model_name)
-spacy_models_list = [nlp_english]
+def init_spacy(language):
```

Comment:
“init_spacy()” function can be moved to infotype_utils.py. Contract of infotype_helper.py is to include the infotype prediction specific logic, other helper methods can reside in infotype_utils.py.